### PR TITLE
Pill Bottle Count Shown on UI

### PR DESCRIPTION
**Pull request by chadleytan**
*Not completely done but main functionality for keeping track of the number of pill bottles held is working. Definitely plan on making the UI look better later and replace the image with the actual throwable object.

Screen Shot 2020-02-26 at 12 25 56 AM

*Also cleaned up `ContinueConversation()` in `DialogueInteractable.cs`

### CODE REVIEW

File: Assets/Scripts/ItemThrowCount.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    TextMeshProUGUI text;
+
+
+    void Start()
+    {
+        playerManager = GameManager.Instance.GetPlayerManager();
+
+        if(playerManager == null)
+        {
+            Debug.LogError("Player manager does not exist");
```
Comment (jurcaua) [inline Assets/Scripts/ItemThrowCount.cs]: Instead can we do `Utils.LogErrorAndExitPlayMode` (I think it's called?)

File: Assets/Scripts/ItemThrowCount.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+{
+    private int numItemsHeld = 0;
+    PlayerManager playerManager;
+    TextMeshProUGUI text;
+
+
+    void Start()
+    {
+        playerManager = GameManager.Instance.GetPlayerManager();
+
```
Comment (jurcaua) [inline Assets/Scripts/ItemThrowCount.cs]: Remove this empty space to connect these two blocks?

File: Assets/Scripts/ItemThrowCount.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        }
+
+        playerManager.OnThrow += HandleOnThrewBottle;
+        playerManager.OnPickup += OnHandlePickup;
+
+        text = Utils.GetRequiredComponentInChildren<TextMeshProUGUI>(this);
+
+        if(text == null)
+        {
+            Debug.LogError("Throwable Object text is null");
```
Comment (jurcaua) [inline Assets/Scripts/ItemThrowCount.cs]: Same as above, `Utils.<that long function name>`?

File: Assets/Scripts/ItemThrowCount.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        }
+        text.SetText("x" + numItemsHeld.ToString());
```
Comment (jurcaua) [inline Assets/Scripts/ItemThrowCount.cs]: Call `DisplayUpdate` here instead to keep it clean

**Review by jurcaua — CHANGES_REQUESTED** [review]
Very nice QUICK TIMES PR EH
logic-wise it's simple and sweet, had some comments for cleanup but otherwise looks CRISP

File: Assets/Scripts/ItemThrowCount.cs (hunk 4 above)
Comment (chadleytan) [reply Assets/Scripts/ItemThrowCount.cs]: Good catch haha

**Review by jurcaua — APPROVED** [review]
boom boom boom quick times 💯

**Pull request merged**